dask_expr/_collection.py

```diff
@@ -1182,6 +1189,99 @@ def nunique(self, axis=0, dropna=True):
                 ]
             )
 
+    def info(self, buf=None, verbose=False, memory_usage=False):
+        """
+        Concise summary of a Dask DataFrame
+        """
+        from dask.utils import M
+
+        mem_use = memory_usage
+
+        if buf is None:
+            import sys
+
+            buf = sys.stdout
+        lines = [str(type(self))]
+
+        if len(self.columns) == 0:
+            lines.append(f"{type(self.index._meta).__name__}: 0 entries")
+            lines.append(f"Empty {type(self).__name__}")
+            if PANDAS_GE_150:
```

Comment:
we don't support pandas < 2.0

Reply:
https://github.com/dask-contrib/dask-expr/pull/513/commits/e918ff965fc46d90bc047cb68fa0ace22394ab87 done.